### DISCUSSION TITLE
ERA5-PDS (2021-2022)

### DISCUSSION THREAD

**Discussion by malmistry** [microsoft/PlanetaryComputer#242] [Data request]
Dear All,
I have been using the planetary computer for processing climate data variables. I am presently interested in the ERA5 reanalysis daily data which I can aggregate from ERA5-hourly. 

It appears that the last year of the data is currently 2020 which is nearly 2-1/2 years less data than the one currently available through Copernicus Climate Data Store (CDS). I was wondering if the data/STAC for 2021 and 2022 can also be added for ERA5.

On a slightly different note, are there any plans to include ERA5-Land as well? I have seen an earlier discussion thread but I am not clear on the final outcome of the discussion.

Thanks and regards,
Malcolm N. Mistry

**Comment by TomAugspurger** [comment]
Hi. Our current ERA5 data is indeed a static copy. 

I did some work a while back to put together a dynamic update pipeline from CDS. Unfortunately, there were some small differences in the data from CDS and what we already had (IIRC, they were just floating-point difference but I wasn't comfortable with just changing the files currently available through our Blob Storage).

Most likely, we'll add a new STAC collection with the updated data and will deprecate the existing STAC collection. I don't have an ETA for that.

https://github.com/microsoft/PlanetaryComputer/discussions/21 is the thread for ERA5-land. No updates to share there, but I'll post there with any updates.

**Reply by malmistry** [reply]
Many thanks @TomAugspurger! Appreciate the effort to make the big data available to the research community.
